web/views/FcLayoutDrawerMap.vue

```diff
@@ -1,20 +1,55 @@
 <template>
-  <v-row
-    class="fill-height"
-    no-gutters>
-    <v-col
-      v-show="drawerOpen"
-      cols="6">
-      <router-view></router-view>
-    </v-col>
-    <v-col :cols="drawerOpen ? 6 : 12">
-      <PaneMap />
-    </v-col>
-  </v-row>
+  <div
+    class="fc-layout-drawer-map fill-height"
+    :class="{
+      'drawer-open': drawerOpen,
+      horizontal: !vertical,
+      vertical
+    }">
+    <div
+      v-if="hasDrawer"
+      class="pane-drawer-toggle elevation-3 font-size-xl"
+      @click="setDrawerOpen(!drawerOpen)">
+      <div class="text-center">
+        <v-icon
+          v-if="vertical"
+          small>
+          {{drawerOpen ? 'mdi-menu-down' : 'mdi-menu-up'}}
```

Comment:
can we just extract this into a computed property and have something like `<v-icon small>{{drawerToggleIcon}}</v-icon>`?